### ISSUE TITLE
Updated screenshots

### ISSUE THREAD

**Issue by vitorgalvao** [chrisgrieser/alfred-neovim-utilities#7]
### Feature Requested

With the new icon, README screenshots are now outdated. When you make new ones, mind providing a couple (just with the Alfred window) for the Gallery?

Version 2.3.0 is still going live, though.

Thank you.

### Checklist

- [X] The feature would be useful to more users than just me.

**Comment by chrisgrieser** [comment]
Right, forgot. Fixed.

**Comment by vitorgalvao** [comment]
To reiterate:

> When you make new ones, mind providing a couple (just with the Alfred window) for the Gallery?

`:h` works for me, but `vp` does not:

```
search-nvim-plugins.js: execution error: Error: Error: sh: line 0: cd: /Users/vm/.local/share/nvim/lazy: No such file or directory (1)
```

**Comment by chrisgrieser** [comment]
> When you make new ones, mind providing a couple (just with the Alfred window) for the Gallery?

Oh sorry, got sidetracked after updating the README screenshots. Here are the ones for the gallery:
![Pasted image 2024-06-13 at 23 15 38@2x](https://github.com/chrisgrieser/alfred-neovim-utilities/assets/73286100/8dce5e91-384b-48ec-b209-7196b214d314)

![Pasted image 2024-06-13 at 23 17 43@2x](https://github.com/chrisgrieser/alfred-neovim-utilities/assets/73286100/a16f627b-3afc-4a83-987c-c02330af1023)

**Comment by chrisgrieser** [comment]
> `:h` works for me, but `vp` does not:
> 
> ```
> search-nvim-plugins.js: execution error: Error: Error: sh: line 0: cd: /Users/vm/.local/share/nvim/lazy: No such file or directory (1)
> ```

Ah, the plugin search was missing a validation that the local plugin path exists before trying to add icons. Fixed in release 2.3.2

**Comment by vitorgalvao** [comment]
Thank you. Updated.
